libobs/util/cf-lexer.c

```diff
@@ -71,22 +71,26 @@ char *cf_literal_to_str(const char *literal, size_t count)
 
 	if (count < 2)
 		return NULL;
-	if (literal[0] != literal[count-1])
+	if (literal[0] != literal[count - 1])
 		return NULL;
 	if (literal[0] != '\"' && literal[0] != '\'')
 		return NULL;
-
-	str = bmalloc(count - 1);
-	temp_src = literal;
+	//bzalloc so that the string is memset w/ 0s
```

Comment:
A styling nitpick -- especially in C code, I prefer not using single-line comments (`//`).  Instead, for your comments, format them like this to keep consistency with the rest of the project:

```C
	/* bzalloc so the string is zeroed */
```

Also, I'd consider this particular comment unnecessary, as anyone who uses C should know fairly quickly that it's a zeroing allocation function.  Please remove this particular comment.

The other two comments however are good to have, because the code is not as obvious.  However, please change them so that they use `/* ` and ` */`, like above.